File: LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentAuthCallback.java
package com.braintreepayments.api;

/**
 * Callback for receiving result of
 * {@link LocalPaymentClient#createPaymentAuthRequest(LocalPaymentRequest, LocalPaymentInternalAuthRequestCallback)}.
 */
public interface LocalPaymentAuthCallback {

    /**
     * @param paymentAuthRequest a request used to launch the PayPal web authentication flow
     */
    void onPayPalPaymentAuthRequest(LocalPaymentAuthRequest paymentAuthRequest);
}

File: LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentAuthCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of
 * [LocalPaymentClient.createPaymentAuthRequest].
 */
fun interface LocalPaymentAuthCallback {
    /**
     * @param paymentAuthRequest a request used to launch the PayPal web authentication flow
     */
    fun onLocalPaymentAuthRequest(paymentAuthRequest: LocalPaymentAuthRequest)
}
